### PR TITLE
Crosswords blind ps version

### PR DESCRIPTION
**Pull request by NathanielBennett**
This is the blind and partially sighted version of the crossword page. So Here, instead of displaying a grid we display a list showing the locations of each BLANK SQUARE by row ( on the left  below ) where the column is represented by a letter so for this crossword:  http://m.code.dev-theguardian.com/crosswords/quick/14115 there are NO blank squares in row 1 (1 as displayed) but blank spaces in B D F H J L on row 2. Also - each clue has the starting co-ordinates (Row-Column) displayed in brackets at the beginning of each line: 

Thus: 

blank-crossword

### CODE REVIEW

**Comment by johnduffell** [comment]
I realise it's probably only in the source code, but possibly NonVisual or Accessible might be more appropriate than Blind here?

**Comment by NathanielBennett** [comment]
Call - Hang on. I'll sort it

File: applications/app/views/blindPsCrossword.scala.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            @fragments.discussionFooter(crosswordPage.isCommentable, crosswordPage.isClosedForComments, crosswordPage.shortUrlId)
+
+            @if(crosswordPage.isCommentable) {
+                <div class="js-repositioned-comments content__repositioned-comments"></div>
+            }
+
+            @fragments.onwardPlaceholder()
+        </div>
+    </div>
+}
```
Comment (johnduffell) [inline applications/app/views/blindPsCrossword.scala.html]: more much commonality is there between the standard and this version?  might be possible to extract most of this into a common template

**Comment by johnduffell** [comment]
this feels like a different way of presenting the same data, Ideally we could use entirely CSS to render an accessible version, but I realise that CSS may not be up to the task.  I do think it would be good to see as much reuse as possible to reduce the chance of breaking this in future though.
As for reviewing the design, I'm not privy to the research done to decide how to present crosswords to screen reader users, but I'm assuming that's all tested and ok =)

**Comment by gklopper** [comment]
Nice to see this in here

**Comment by NathanielBennett** [comment]
@johnduffell - OK everything's accessible now. Point taken about about data representation - it's difficult to see though how CSS could render a SVG on the onehand and a what's essentially a list of characters on the other. Templates deduped and fragmentated as much as could. N

**Comment by johnduffell** [comment]
cool, that's great! :+1: ship it and see what the users think!

**Comment by OliverJAsh** [comment]
We should try and make the normal version accessible at some point, instead of having a separate page.

**Pull request merged**